templates/webpack-default/README.md

```diff
@@ -2,17 +2,17 @@
 
 Here's how your Alloy directory is laid out.
 
-models              your model files go here
-controllers         your controllers files go here
-views               yep, the views go here. you're getting it
-styles              your style (.tss) files for your views go here
-assets              All files here will be deployed into Resources
-lib                 put your own libraries here and use require('@/name') to load it
-migrations          generated model migrations go here
-widgets             pre-built, reusable components for your Alloy apps.
+models your model files go here
```

Comment:
This whole section is unreadable. Consider a list with the directory name in code backticks followed by a dash before the description.